EvenTask/Helpers/String+InteractorKeys.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  String+InteractorKeys.swift
+//  EvenTask
+//
+//  Created by Ahmed Ramy on 12/7/18.
+//  Copyright © 2018 Ahmed Meguid. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    // MARK:- Login Interactor
+    static let emailKey = "Email"
+    static let passwordKey = "Password"
+    
+    // MARK:- AddEvent Interactor
+    static let eventTitleKey = "title"
+    static let eventDescriptionKey = "description"
+    static let eventPriorityKey = "priority"
+    static let eventisPermanentKey = "permanent"
+    static let eventUserIDKey = "userId"
+}
```

Comment:
We send this keys  to the validator to append it in the error message, so its purpose is only logging, so you're right we need to separate (for localization in the future).
My comment, is to collect all the fields/inputs strings in one file with this name.
- rename the file
- remove the MARK
- make it simple and general (for ex: emailField = "Email"
- I think email/password should be all lowercase (my bad)